### PR TITLE
Consortium page.

### PR DESCRIPTION
**Pull request by maximedenes**

### CODE REVIEW

**Comment by Zimmi48** [comment]
About time!

File: pages/consortium.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+
+    </p>
+    <p>Partners declare a number of intensive users of Coq they represent, and the Consortium recommends a corresponding level of
+        service (a number of engineer hours, which can be used for support or training). The cost corresponds to that specific level of service. Academic members receive a 50% discount.</p>
+    <table>
+        <tr><td>Users</td><td>Engineer hours</td><td>1 year</td><td>2 years</td><td>3 years</td></tr>
+        <tr>
+            <td>
+                5-7
```
Comment (Zimmi48) [inline pages/consortium.html]: Is it really wanted that the first line, which is per user, be restricted to 5-7 users and not to any size < 8 users?

File: pages/consortium.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+for the Coq Consortium to receive donations or establish specific support contracts.</p>
+</div>
+
+</div>
+
+<div class="framework">
+<div class="frameworklabel">How to join?</div>
+<div class="frameworkcontent">
+    <p>The Coq Consortium is currently being created by the Inria Foundation.
+        If you are interested, <please contact <a href="mailto:yves.bertot@inria.fr">Yves Bertot</a>
```
Comment (Zimmi48) [inline pages/consortium.html]: Typo: this `<` is preventing the printing of "please contact".

File: pages/consortium.html [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+<div class="framework">
+<div class="frameworklabel">A pool of resources and a platform for exchanges</div>
+<div class="frameworkcontent">
+
+<p>The objective of the Consortium is to make the community of users feel empowered with respect to
+    the future of the Coq system, giving them opportunities to see how the system will evolve, where it is
+    going, and to take an active part in this process. On the one hand, we wish to establish <strong>transparent
+    ways of contributing financially and technically to the development of the system</strong>; on the other
+    hand we wish to <strong>make sure that the most pressing needs of the user community are addressed
+    efficiently</strong> by providing a fantastic working environment to some of the most competent developers
```
Comment (Zimmi48) [inline pages/consortium.html]: It looks visually like too much bold to me. Maybe try using italic, or emphasizing less words?

File: incl/header.html [hunk 4]
```diff
@@ -21,6 +21,7 @@
           <li><a href="/download">Get Coq</a></li>
           <li><a href="/documentation">Documentation</a></li>
           <li><a href="/community">Community</a></li>
+          <li><a href="/consortium">Consortium</a></li>
```
Comment (Zimmi48) [inline incl/header.html]: You should also add a section in the homepage about the Consortium. What is shown on the homepage and what appears in the top bar should be kept in sync.

File: pages/consortium.html (hunk 1 above)
Comment (maximedenes) [reply pages/consortium.html]: Yes

File: pages/consortium.html (hunk 3 above)
Comment (maximedenes) [reply pages/consortium.html]: It looks fine to me.

File: pages/consortium.html (hunk 2 above)
Comment (maximedenes) [reply pages/consortium.html]: Thanks!

File: incl/header.html (hunk 4 above)
Comment (maximedenes) [reply incl/header.html]: Well, it is not in sync today (no "news" item on the main page), and I'd rather add an entry only once we clarify a few things about the consortium structure with Yves. This page is rather meant for people actively looking for the Consortium (through Google) or for people needing to send a pointer (to their funding agency, for example).

Comment (Zimmi48) [reply incl/header.html]: The news still have a specific section on the main page (and all the other pages).

File: pages/consortium.html (hunk 3 above)
Comment (Zimmi48) [reply pages/consortium.html]: Maybe show it to @ybertot in order to get a second opinion.

File: incl/header.html (hunk 4 above)
Comment (maximedenes) [reply incl/header.html]: Ok, but the second point remains.

File: pages/consortium.html (hunk 3 above)
Comment (maximedenes) [reply pages/consortium.html]: Ok, I reduced a bit the number of words in bold.

File: incl/header.html (hunk 4 above)
Comment (Zimmi48) [reply incl/header.html]: In the meantime I would then add it in some place that is less obvious. But that's up to you.

File: pages/consortium.html [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+<div class="frameworklabel">A pool of resources and a platform for exchanges</div>
+<div class="frameworkcontent">
+
+<p>The objective of the Consortium is to make the community of users feel empowered with respect to
+    the future of the Coq system, giving them opportunities to see how the system will evolve, where it is
+    going, and to take an active part in this process. On the one hand, we wish to establish <strong>transparent
+    ways of contributing financially and technically</strong> to the development of the system; on the other
+    hand we wish to make sure that the <strong>most pressing needs of the user community are addressed
+    efficiently</strong> by providing a fantastic working environment to some of the most competent developers
+    trained in our teams.</p>
```
Comment (ybertot) [inline pages/consortium.html]: Although this is not explicit, I have this impression is to centered on inria based teams.  I propose changing to
"to competent developers"

File: pages/consortium.html [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+<div class="framework">
+<div class="frameworklabel">Support services</div>
+<div class="frameworkcontent">
+    <p>The Consortium provides a package of support services:
+        <ul>
+            <li>Premium support for bugs. Reports by Consortium members will be processed in priority by
+the Consortium engineers. Private bug reports can be processed by Consortium engineers
+                under an NDA to explore a code base and minimize a bug example.</li>
+            <li>A session of on-site training each year. (Optional, travel and lodging expenses should be paid
+                by the member).</li>
```
Comment (ybertot) [inline pages/consortium.html]: I believe "accommodation" will be more idiomatic than "lodging"

Comment (aspiwack) [reply pages/consortium.html]: I'd rather have confirmation from native speakers, but it sounds to me as a difference between American and British English.

Comment (maximedenes) [reply pages/consortium.html]: This was written in Andrew's letter, so I'm pretty sure it is idiomatic English :)

File: pages/consortium.html (hunk 5 above)
Comment (maximedenes) [reply pages/consortium.html]: I'm not sure I understand exactly the proposal. Is it only to remove "most"? Or also "trained in our teams"?

**Comment by mattam82** [comment]
Maybe add a link on the index page somewhere as well? Like another "tab link" to the Community paragraph?

File: pages/consortium.html (hunk 5 above)
Comment (ybertot) [reply pages/consortium.html]: remove "trained in our teams"

File: pages/consortium.html (hunk 6 above)
Comment (ybertot) [reply pages/consortium.html]: okay then

**Pull request merged**